### PR TITLE
Small fix 

### PR DESCRIPTION
**Pull request by jontofront**
device_class change from SPEED to POWER_FACTOR

### CODE REVIEW

**Comment by pblxptr** [comment]
Hi, thanks I will check it. In the meantime please provide a broader description because from what I see there are more changes than only device class change.

File: custom_components/econet300/sensor.py
Comment (jontofront) [inline custom_components/econet300/sensor.py:1]: Add POWER_FACTOR factor by Documentation
image
https://developers.home-assistant.io/docs/core/entity/sensor/

File: custom_components/econet300/sensor.py [hunk 1]
```diff
@@ -41,7 +41,7 @@ class EconetSensorEntityDescription(SensorEntityDescription):
         icon="mdi:fan",
         native_unit_of_measurement=PERCENTAGE,
         state_class=SensorStateClass.MEASUREMENT,
-        device_class=SensorDeviceClass.SPEED,
+        device_class=SensorDeviceClass.POWER_FACTOR,
```
Comment (jontofront) [inline custom_components/econet300/sensor.py]: There was warning in logs Generic speed can be measure ft/s, in/d, in/h, km/h, kn, m/s, mph, mm/d but not in %

File: custom_components/econet300/binary_sensor.py [hunk 2]
```diff
@@ -31,34 +33,69 @@ class EconetBinarySensorEntityDescription(BinarySensorEntityDescription):
...
     ),
     EconetBinarySensorEntityDescription(
         availability_key="lighter",
         key="lighterWorks",
         name="Lighter",
         icon="mdi:fire",
+        icon_off="mdi:fire-off",
+        device_class=BinarySensorDeviceClass.RUNNING
+    ),
+    EconetBinarySensorEntityDescription(
```
Comment (jontofront) [inline custom_components/econet300/binary_sensor.py:79]: add feeder binary sensor

File: custom_components/econet300/binary_sensor.py [hunk 3]
```diff
@@ -31,34 +33,69 @@ class EconetBinarySensorEntityDescription(BinarySensorEntityDescription):
...
+        key="fanWorks",
+        name="Feeder",
+        icon="mdi:screw-lag",
+        device_class=BinarySensorDeviceClass.RUNNING
+    ),
+        EconetBinarySensorEntityDescription(
+        availability_key="fan",
+        key="fan2ExhaustWorks",
+        name="Feeder",
+        icon="mdi:fan",
```
Comment (jontofront) [inline custom_components/econet300/binary_sensor.py:97]: my bad I see my mistake with key when I make copy paste, sorry

File: custom_components/econet300/binary_sensor.py [hunk 4]
```diff
@@ -31,34 +33,69 @@ class EconetBinarySensorEntityDescription(BinarySensorEntityDescription):
...
     EconetBinarySensorEntityDescription(
         availability_key="pumpSolar",
         key="pumpSolarWorks",
         name="Solar pump",
         icon="mdi:pump",
+        icon_off="mdi:pump-off",
+        device_class=BinarySensorDeviceClass.RUNNING
+    ),
+    EconetBinarySensorEntityDescription(
+        availability_key="pumpCO",
```
Comment (jontofront) [inline custom_components/econet300/binary_sensor.py:64]: I don't know how to name it, It boiler pump
image

File: custom_components/econet300/binary_sensor.py (hunk 3 above)
Comment (jontofront) [reply custom_components/econet300/binary_sensor.py:97]: EconetBinarySensorEntityDescription(
        availability_key="fan",
        key="fanWorks",
        name="Fan", 
        icon="mdi:fan",
        device_class=BinarySensorDeviceClass.RUNNING
    ),

exapmle know in my case:
fan:  false
fanWorks : false


EconetBinarySensorEntityDescription(
        availability_key="fan2Exhaust",
        key="fan2ExhaustWorks", 
        name="Fan2",
        icon="mdi:fan",
        icon_off = "mdi:fan-off",
        device_class=BinarySensorDeviceClass.RUNNING
    )


exapmle know in my case:
fan2Exhaust: false
fan2ExhaustWorks : false

**Comment by jontofront** [comment]
> Hi, thanks I will check it. In the meantime please provide a broader description because from what I see there are more changes than only device class change.

Sorry another PR was create aumaticly when I commit to our branch

**Comment by jontofront** [comment]
Hi,
I don't know why the merge closed, I have working changes in my master which i testing in my HA. 
Can I make new merge request?

**Pull request closed without merging**